examples/powertools-examples-cloudformation/src/main/java/helloworld/App.java

```diff
@@ -0,0 +1,166 @@
+package helloworld;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.events.CloudFormationCustomResourceEvent;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import software.amazon.awssdk.awscore.exception.AwsServiceException;
+import software.amazon.awssdk.core.exception.SdkClientException;
+import software.amazon.awssdk.core.waiters.WaiterResponse;
+import software.amazon.awssdk.http.apache.ApacheHttpClient;
+import software.amazon.awssdk.services.s3.S3Client;
+import software.amazon.awssdk.services.s3.model.*;
+import software.amazon.awssdk.services.s3.waiters.S3Waiter;
+import software.amazon.lambda.powertools.cloudformation.AbstractCustomResourceHandler;
+import software.amazon.lambda.powertools.cloudformation.Response;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Handler for requests to Lambda function.
+ */
+
+public class App extends AbstractCustomResourceHandler {
+    public static final String EXPIRE_OBJECTS_ID = "expire-all-objects-after-x-days";
+    private final static Logger log = LogManager.getLogger(App.class);
+    private final S3Client s3Client;
+
+    public App() {
+        super();
+        s3Client = S3Client.builder().httpClientBuilder(ApacheHttpClient.builder()).build();
+    }
+
+    @Override
+    protected Response create(CloudFormationCustomResourceEvent cloudFormationCustomResourceEvent, Context context) {
+        Objects.requireNonNull(cloudFormationCustomResourceEvent, "cloudFormationCustomResourceEvent cannot be null.");
+        Objects.requireNonNull(cloudFormationCustomResourceEvent.getResourceProperties().get("BucketName"), "BucketName cannot be null.");
+        Objects.requireNonNull(cloudFormationCustomResourceEvent.getResourceProperties().get("RetentionDays"), "RetentionDays cannot be null.");
+
+        log.info(cloudFormationCustomResourceEvent);
+        String bucketName = (String) cloudFormationCustomResourceEvent.getResourceProperties().get("BucketName");
+        log.info("Bucket Name {}", bucketName);
+        int retentionDays = Integer.parseInt(String.valueOf(cloudFormationCustomResourceEvent.getResourceProperties().get("RetentionDays")));
```

Comment:
do we need the String.valueOf ?

Reply:
Yes, because `getResourceProperties` returns a `Map<String, Object>` and `Integer.parseInt` requires a String parameter